### ISSUE TITLE
Implement verdi work plugins

### ISSUE THREAD

**Issue by sphuber** [aiidateam/aiida_core#1048]
In parallel with `verdi calculation plugins` it would be nice to have a command for `verdi work` that would show the registered workflow entry points. When an entry point is passed as an argument, the detailed information of that workflow should be shown. We will only implement this for new style workflows, i.e. `WorkChain` based classes.
